integration-tests/src/lib.rs

```diff
@@ -139,5 +148,66 @@ pub struct CompiledContract {
     pub bin_runtime: Bytes,
 }
 
+/// Build circuit input builder for a block
+pub async fn build_circuit_input_builder_block(block_num: u64) {
+    let cli = get_client();
+    let cli = BuilderClient::new(
+        cli,
+        CircuitsParams {
+            max_rws: 800000,
+            max_txs: 10,
+            max_calldata: 4000,
+            max_bytecode: 4000,
+            max_copy_rows: 800000,
+            max_evm_rows: 0,
+            max_exp_steps: 1000,
+            max_keccak_rows: 0,
+        },
+    )
+    .await
+    .unwrap();
+
+    // 1. Query geth for Block, Txs and TxExecTraces
+    let (eth_block, geth_trace, history_hashes, prev_state_root) =
+        cli.get_block(block_num).await.unwrap();
+
+    // 2. Get State Accesses from TxExecTraces
+    let access_set = get_state_accesses(&eth_block, &geth_trace).unwrap();
+    trace!("AccessSet: {:#?}", access_set);
+
+    // 3. Query geth for all accounts, storage keys, and codes from Accesses
+    let (proofs, codes) = cli.get_state(block_num, access_set).await.unwrap();
+
+    // 4. Build a partial StateDB from step 3
+    let (state_db, code_db) = build_state_code_db(proofs, codes);
+    trace!("StateDB: {:#?}", state_db);
+
+    // 5. For each step in TxExecTraces, gen the associated ops and state
+    // circuit inputs
+    let builder = cli
+        .gen_inputs_from_state(
+            state_db,
+            code_db,
+            &eth_block,
+            &geth_trace,
+            history_hashes,
+            prev_state_root,
+        )
+        .unwrap();
+
+    trace!("CircuitInputBuilder: {:#?}", builder);
+}
+
+lazy_static! {
+    /// The block that has only one anchor
+    pub static ref TAIKO_BLOCK_ANCHOR_ONLY: u64 = 3261;
+    /// The block that has propose block contract call
+    pub static ref TAIKO_BLOCK_PROPOSE_BLOCK: u64 = 3209;
+    /// The block that has prove block contract call
+    pub static ref TAIKO_BLOCK_PROVE_BLOCK: u64 = 3211;
+    /// The block that has ERC20 transfer
+    pub static ref TAIKO_BLOCK_TRANSFER_SUCCEED: u64 = 3347;
+}
```

Comment:
I don't think this needs `lazy_static!`, these can just be normal const values.

Reply:
Updated to use normal const values